form.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require_once 'connectToDb.php';
+require_once 'functions.php';
+
+if (count($_POST) === 5) {
+    if (array_search('', $_POST)) {
+        header('Location: index.php?error=Registration%20failed');
+        exit;
+    }
+    $db = getDb();
+
+    $title = $_POST['title'];
+    $artist = $_POST['artist'];
+    $movement = $_POST['movement'];
+    $year = $_POST['year'];
+    $image = $_POST['image'];
```

Comment:
I would suggest performing some basic validation against these values - even something as simple as running them through trim() and htmlspecialchars() to make sure all is in order before you insert them.

Users are generally idiots so it's quite common that you'll get an input like this 'Thing         ' with loads of random spaces on the end etc